homework_07.py

```diff
@@ -0,0 +1,31 @@
+# Створити телефонну книгу,
+phonebook = {
+    'Anna':'+111111111',
+    'Bogdan':'+222222222',
+    'Vova':'+333333333',
+    'Gena':'+444444444'
+}
+
+# яка матиме команди:
+command = ''
+while command != 'exit':
+    command = input("Enter a command stats/add/delete/list/show or exit: ")
+    match command:
+        case 'stats': # stats: кількість записів
+            key_list = phonebook.keys()
+            print(len(key_list))
+        case 'add': # add: додати запис
+            name = input("Enter a name: ")
+            phonenumber = input("Enter a phone number: ")
+            phonebook[name] = phonenumber
```

Comment:
It is not allowed by the task to override records. Please, make additional check here